plugins/explore/README.md

```diff
@@ -104,11 +104,11 @@ Create a custom explore page in
 
 ```tsx
 import {
-  DomainExplorerContent,
+  CatalogKindExplorerContent,
   ExploreLayout,
 } from '@backstage/plugin-explore';
 import React from 'react';
-import { InnserSourceExplorerContent } from './InnserSourceExplorerContent';
+import { AcmeInnserSourceExploreContent } from './InnserSourceExplorerContent';
```

Comment:
These were meant to say "Inner", not "Innser". Same below.

If you add Acme (not really necessary, right?) you should do it to the filename to the right too

Reply:
I only added Acme here in the import because that was the component that was used below, but I would also be happy to remove it from both places.